src/api.v2/controllers/accessController.js

```diff
@@ -1,19 +1,62 @@
-const getUserRoles = require('../helpers/access/getUserRoles');
+const getExperimentUsers = require('../helpers/access/getExperimentUsers');
+const createUserInvite = require('../helpers/access/createUserInvite');
+const removeAccess = require('../helpers/access/removeAccess');
+const postRegistrationHandler = require('../helpers/access/postRegistrationHandler');
+
+const OK = require('../../utils/responses/OK');
 const getLogger = require('../../utils/getLogger');
 
 const logger = getLogger('[AccessController] - ');
 
-const getExperimentUsers = async (req, res) => {
+const getUserAccess = async (req, res) => {
   const { experimentId } = req.params;
 
   logger.log(`Fetching users for experiment ${experimentId}`);
-  const users = await getUserRoles(experimentId);
+  const users = await getExperimentUsers(experimentId);
 
   logger.log(`Users fetched for experiment ${experimentId}`);
 
   res.json(users);
 };
 
+const inviteUser = async (req, res) => {
+  const { experimentId } = req.params;
+  const {
+    userEmail, role,
+  } = req.body;
+
+  logger.log(`Inviting users to experiment ${experimentId}`);
+  await createUserInvite(experimentId, userEmail, role, req.user);
+
+  logger.log(`Users invited to experiment ${experimentId}`);
+
+  res.json(OK());
+};
+
+const revokeAccess = async (req, res) => {
+  const { experimentId } = req.params;
+  const { userEmail } = req.body;
+
+  logger.log(`Deleting user access from experiment ${experimentId}`);
```

Comment:
in here, you are removing access for only 1 experiment and only 1 user, right? In this case, can you also add the user email in the log?

Reply:
Yes. Access is removed one at a time.

Reply:
can you add the email for which you are removing access in the logs in this case?